### DISCUSSION TITLE
Can I set up Traps with a Raspberry Pi and the Onboard Wireless Adapter?

### DISCUSSION THREAD

**Discussion by Kryptach** [nzymedefense/nzyme#549] [Questions] [answered]
I currently only have one external USB Adapter, which is running as the main device in Monitor Mode to run nzyme. So now my question: Can I use the Onboard Wireless Adapter on my Raspberry Pi for setting up traps or do I need an external Adapter with Monitor Mode?

**Comment by lennartkoopmann** [comment, marked as answer]
Hi! The easiest is to get a second adapter, but you should also check out the nexmon drivers for RPI: https://github.com/seemoo-lab/nexmon
